api/src/queries/project-queries.test.ts

```diff
@@ -14,38 +14,75 @@ describe('postProjectSQL', () => {
+    };
+
+    const coordinatorData = {
+      first_name: 'coordinator_first_name',
+      last_name: 'coordinator_last_name',
+      email_address: 'coordinator_email_address@email.com',
+      coordinator_agency: 'coordinator_agency_name',
+      share_contact_details: false
+    };
+
+    const locationData = {
+      location_description: 'a location description',
+      regions: ['Valid Region']
+    };
+
+    const postProjectData = new PostProjectData(projectData);
+    const postCoordinatorData = new PostCoordinatorData(coordinatorData);
+
     it('returns a SQLStatement', () => {
-      const projectData = {
-        name: 'name_test_data',
-        objectives: 'objectives_test_data',
-        scientific_collection_permit_number: 'scientific_collection_permit_number_test_data',
-        management_recovery_action: 'management_recovery_action_test_data',
-        start_date: 'start_date_test_data',
-        end_date: 'end_date_test_data',
-        caveats: 'caveats_test_data',
-        comments: 'comments_test_data'
-      };
+      const postLocationData = new PostLocationData(locationData);
+      const response = postProjectSQL({ ...postProjectData, ...postCoordinatorData, ...postLocationData });
 
-      const coordinatorData = {
-        first_name: 'coordinator_first_name',
-        last_name: 'coordinator_last_name',
-        email_address: 'coordinator_email_address@email.com',
-        coordinator_agency: 'coordinator_agency_name',
-        share_contact_details: false
-      };
+      expect(response).to.not.be.null;
+    });
 
-      const locationData = {
-        location_description: 'a location description',
-        regions: ['Valid Region']
+    it('returns a SQLStatment with the geometries inserted correctly', () => {
+      const locationDataWithGeo = {
+        ...locationData,
+        geometry: [
+          {
+            type: 'Feature',
+            id: 'myGeo1',
+            geometry: {
+              type: 'Polygon',
+              coordinates: [
+                [
+                  [-128, 55],
+                  [-128, 55.5],
+                  [-128, 56],
+                  [-126, 58],
+                  [-128, 55]
+                ]
+              ]
+            },
+            properties: {
+              name: 'Biohub Islands'
+            }
+          }
+        ]
       };
 
-      const postProjectData = new PostProjectData(projectData);
-      const postCoordinatorData = new PostCoordinatorData(coordinatorData);
-      const postLocationData = new PostLocationData(locationData);
-
+      const postLocationData = new PostLocationData(locationDataWithGeo);
       const response = postProjectSQL({ ...postProjectData, ...postCoordinatorData, ...postLocationData });
 
       expect(response).to.not.be.null;
+      expect(
+        response?.values.includes(
+          '{"type":"Polygon","coordinates":[[[-128,55],[-128,55.5],[-128,56],[-126,58],[-128,55]]]}'
+        )
+      ).to.not.be.null;
```

Comment:
You can probably leverage the `deep` equality tester if you want to assert that the content matches exactly: https://www.chaijs.com/api/bdd/#method_deep

Reply:
Otherwise I think this always passes? `expect(boolean).to.not.be.null` Unless false is considered a null type value

Reply:
thank you! nice catch :)